---
title: "Genomic landscapes and fitness ridges"
subtitle: "across the speciation continuum"
author: "Peter Ralph"
date: "College de France // May 2019"
---





# Genomic landscapes

##


![Langley et al 2012](figs/from_the_literature/langley-et-al-2012-chr3-pi-and-rho.png)


<!--
## *Homo sapiens*

![Hernandez et al 2011](figs/from_the_literature/hernandez-2011-dip-in-diversity.png)

*Hernandez et al 2011*
-->

## Diversity correlates with recombination rate

![Corbett-Detig et al](figs/from_the_literature/corbett-detig-divergence-recomb-drosophila.png)

Corbett-Detig et al 2015; Cutter & Payseur 2013 

--------------------

![Corbett-Detig et al](figs/from_the_literature/corbett-detig-divergence-recomb-all-species.png){width=70%}

Corbett-Detig et al 2015

<!--
-------------------

![McVicker et al 2009](figs/from_the_literature/mcvicker-2009-hu_chr1-landscape.png)
![](figs/from_the_literature/mcvicker-2009-hu_chr1-landscape-caption.png)

McVicker et al 2009
-->

##

What causes variation in genetic diversity?


##

$$
\begin{aligned}
\pi &= \text{ (within-pop diversity) } \\
d_{xy} &= \text{ (between-pop divergence) } \\
F_{ST} &= \text{ (relative differentiation) }
\end{aligned}
$$

::: {.centered}

![](figs/fst_tree.png){width=60%}

:::

<!-- section THEORY -->

# Linked selection, in theory

--------------

linked selection

: The indirect effects of selection on genomic locations
  that are *linked* to the sites under selection by a lack of recombination.

::: {.centered}

![](figs/sweep_haps.png){.fragment width="40%"}
![](figs/background_haps.png){.fragment width="40%"}

:::

--------------------

**Linked Selection: a field guide**

*composed of rough sketches*

. . .

*citations: numerous*

0. Neutrality
-------------

![](figs/linked-selection-guide/linked-selection-site-neutral_early.png)

0. Neutrality
-------------

![](figs/linked-selection-guide/linked-selection-site-neutral.png)


1. A "hard" sweep
-----------------

![](figs/linked-selection-guide/linked-selection-site-hard.png)

2. A "soft" sweep
-----------------

![](figs/linked-selection-guide/linked-selection-site-soft.png)

3. Adaptive introgression
-------------------------

![](figs/linked-selection-guide/linked-selection-site-introgression.png)

4. Local adaptation with gene flow
-------------------

![](figs/linked-selection-guide/linked-selection-site-local_adaptation.png)

<!-- [Bierne 2010](https://doi.org/10.1111%2fj.1558-5646.2010.01050.x) -->

5. Background selection
-----------------------

![](figs/linked-selection-guide/linked-selection-region-background.png)

6. Recurrent sweeps
-------------------

![](figs/linked-selection-guide/linked-selection-region-recurrent_sweeps.png)

7. Recurrent introgression
--------------------------

![](figs/linked-selection-guide/linked-selection-region-recurrent_introgression.png)

## Etcetera

8. Recent balanching selection

9. Long-term balancing selection

10. Recurrent local adaptation after a split

11. Recurrent, transient balancing selection

12. All of the above?


<!-- AURENTIACUS -->

# How do genetic landscapes change over time?


The *Mimulus aurantiacus* species complex
-----------------------------------------

::: {.centered}
![](figs/aurantiacus/phylogeny.png)
:::


------------------

::: {.centered}
![](figs/aurantiacus/stankowski-color-cline.png){width=70%}
:::

----------------------------------

![https://www.biorxiv.org/content/early/2018/06/21/342352](figs/aurantiacus/preprint.png)

----------------------------------

![https://www.biorxiv.org/content/early/2018/06/21/342352](figs/aurantiacus/preprint_peeps.png)



-------------------------

The data:

- chromosome-level genome assembly
- $20\times$ coverage of 8 taxa and outgroup (*M.clevelandii*)
- diversity ($\pi$) and divergence ($d_{xy}$) in windows
- 36 pairwise comparisons among 9 taxa
- estimates of recombination rate and gene density
    from map and annotation

<!--
## A spectrum of differentiation

![](figs/aurantiacus/distributions.png)
-->

# Landscapes of bush monkeyflower

-------------------

::: {.centered}
![](figs/aurantiacus/lg1.png){ width="46%" }
![](figs/aurantiacus/lg3.png){ width="47%" }
:::

-------------------

::: {.centered}
![](figs/aurantiacus/just_phylogeny.png){ width="47%" }
![](figs/aurantiacus/pi_cor.png){ width="46%" }
:::


-------------------

::: {.centered}
![](figs/aurantiacus/just_phylogeny.png){ width="60%" }
:::



---------------------

<!--
![](figs/aurantiacus/emergence_of_fst_0.png)
![](figs/aurantiacus/emergence_of_fst_1.png)
![](figs/aurantiacus/emergence_of_fst_2.png)
![](figs/aurantiacus/PUN_AUS_fst.png)
![](figs/aurantiacus/PUN_CAL_fst.png)
![](figs/aurantiacus/PUN_LON_fst.png)
![](figs/aurantiacus/PUN_AUR_fst.png)
![](figs/aurantiacus/PUN_FLE_fst.png)
![](figs/aurantiacus/PUN_ARI_fst.png)
![](figs/aurantiacus/PUN_BIF_fst.png)
![](figs/aurantiacus/PUN_CLV_fst.png)
![](figs/aurantiacus/cor-across-time-0.png)
![](figs/aurantiacus/cor-across-time-1.png)
![](figs/aurantiacus/cor-across-time-2.png)
-->

:::: {.columns}
:::::::: {.column width=80%}

![](figs/aurantiacus/rising_landscapes/divergence_by_node_puniceus_LG3.png)

::::
:::::::: {.column width=20%}

![](figs/aurantiacus/labeled_phylogeny_puniceus.png){width="250%"}

::::
::::::::

---------------------

:::: {.columns}
:::::::: {.column width=80%}

![](figs/aurantiacus/rising_landscapes/divergence_by_node_long-caly_LG3.png)

::::
:::::::: {.column width=20%}

![](figs/aurantiacus/labeled_phylogeny_long-caly.png){width="250%"}

::::
::::::::

---------------------

:::: {.columns}
:::::::: {.column width=80%}

![](figs/aurantiacus/rising_landscapes/divergence_by_node_caly_LG3.png)

::::
:::::::: {.column width=20%}

![](figs/aurantiacus/labeled_phylogeny_caly.png){width="250%"}

::::
::::::::

---------------------

:::: {.columns}
:::::::: {.column width=80%}

![](figs/aurantiacus/rising_landscapes/divergence_by_node_aura_LG3.png)

::::
:::::::: {.column width=20%}

![](figs/aurantiacus/labeled_phylogeny_aura.png){width="250%"}

::::
::::::::

<!--
---------------------

:::: {.columns}
:::::::: {.column width=80%}

![](figs/aurantiacus/rising_landscapes/divergence_by_node_arid-parv_LG3.png)

::::
:::::::: {.column width=20%}

![](figs/aurantiacus/labeled_phylogeny_arid-parv.png){width="250%"}

::::
::::::::
-->

---------------------

:::: {.columns}
:::::::: {.column width=80%}

![](figs/aurantiacus/rising_landscapes/divergence_by_node_parv_LG3.png)

::::
:::::::: {.column width=20%}

![](figs/aurantiacus/labeled_phylogeny_parv.png){width="250%"}

::::
::::::::

---------------------

:::: {.columns}
:::::::: {.column width=80%}

![](figs/aurantiacus/rising_landscapes/divergence_by_node_gran_LG3.png)

::::
:::::::: {.column width=20%}

![](figs/aurantiacus/labeled_phylogeny_gran.png){width="250%"}

::::
::::::::

---------------------

:::: {.columns}
:::::::: {.column width=80%}

![](figs/aurantiacus/rising_landscapes/divergence_by_node_all_LG3.png)

::::
:::::::: {.column width=20%}

![](figs/aurantiacus/labeled_phylogeny_all.png){width="250%"}

::::
::::::::

--------------------

![](figs/aurantiacus/divergence_by_node_LG3.png)

# Conclusions

## Observations

> 1. $d_{xy}$ is similar but not the same for different pairs of taxa separated by the same node in the phylogeny.
> 
>     - ![Rightarrow](figs/finger_right.png){.finger width="15%"} some early introgression
> 
> 2. At first, $d_{xy}$ is like $\pi$ but shifted up.
> 
>     - ![Rightarrow](figs/finger_right.png){.finger width="15%"}  good.
> 
> 3. But then $d_{xy}$ starts to flatten out.
> 
>     - ![Rightarrow](figs/finger_right.png){.finger width="15%"}  increased *fixation* rate due to positive mutation?
>     - or, something about repeats?



##


- Evolution of the landscape of diversity across $\approx$ 1.5 million years!

- Shared targets of linked selection across taxa?

. . .

- But, what *kind* of linked selection? \
    How strongly and frequently does it act? \
    On how many targets?

. . .

Consequences for: genetic load/disease,
speed of evolution,
trait architecture.


# Network drift

## 

:::: {.columns}
:::::::: {.column width=50%}


The data suggest **lots** of selection.


What kind? Is this plausible?


![Rightarrow](figs/finger_right.png){.finger width="15%"}
We need explicit models.

::::
:::::::: {.column width=50%}

![Josh](figs/kalman_walk/jsschiffman.jpg)

Josh Schiffman

::::
::::::::

## 

::: {.columns}
::::::: {.column width="50%"}

At time $t$,

$\kappa(t)$ : internal state \
    ($n$-vector of expression levels)

$\phi(t)$ : $k$-vector of phenotype

$u(t)$ : $\ell$-vector of input signals

:::
::::::: {.column width="50%"}

$$\begin{aligned}
    \dot \kappa(t) &= A \kappa(t) + B u(t) \\
    \phi(t) &= C \kappa(t) 
\end{aligned}$$

where

$A$ : $n \times n$ matrix of interactions

$B$ : $n \times \ell$ input to internal state

$C$ : $k \times n$ internal state to output


:::
:::::::

## 


If $\phi(0) = 0$,
$$\begin{aligned}
    \phi(t) = \int_0^t  h(t-s) u(s) ds .
\end{aligned}$$
where
$$\begin{aligned}
    h(s) = C e^{sA} B
\end{aligned}$$


## Are systems unique?

How many *systems* $(A, B, C)$
have the *same* input-output behavior,
$$\begin{aligned}
    u \mapsto \phi,
\end{aligned}$$
if any?

. . .

Answer: generically, **lots** (Kalman 1963).

## Example: two-gene oscillator

:::: {.columns}
:::::::: {.column width="50%"}

$$\begin{aligned}
    A &= \begin{bmatrix}
    0 & 1 \\
    -1 & 0
    \end{bmatrix} \\
    B &= \begin{bmatrix}
    1 \\ 1 \end{bmatrix} \\
    C &= \begin{bmatrix}
    1 & 0 \end{bmatrix}
\end{aligned}$$

![](figs/kalman_walk/osc_impulse.png){ width="80%" }

::::
:::::::: {.column width=50%}

![](figs/kalman_walk/oscillator_1.png){ width="70%" }

::::
::::::::


## Example: two-gene oscillator

:::: {.columns}
:::::::: {.column width="50%"}

$$\begin{aligned}
    A &= \begin{bmatrix}
    1 + \frac{1}{\tau - 1} & 2 \tau + \frac{1}{\tau - 1} \\
    \frac{-1}{\tau - 1} & - 1 - \frac{1}{\tau - 1}
    \end{bmatrix} \\
    B &= \begin{bmatrix}
    1 \\ 1 \end{bmatrix} \\
    C &= \begin{bmatrix}
    1 & 0 \end{bmatrix}
\end{aligned}$$

::::
:::::::: {.column width=50%}

![](figs/kalman_walk/oscillator_2.png)

::::
::::::::

## Example: two-gene oscillator


::: {.centered}
![](figs/kalman_walk/oscillator_3.png){width="70%"}
:::

## Consequences for speciation?

::: {.centered}
![](figs/kalman_walk/conceptual_fig.png){width="100%"}
:::


# Summary

-------

> - Landscapes of genetic diversity are clues to the ongoing types and intensities
    of natural selection.

> - We can see them emerge, and change with time.

> - What makes these landscapes? Still unknown.

> - Explicit models can provide surprising predictions.



Thanks!
-------

Mimulus: Madeline Chase, Sean Stankowski, Matt Streisfeld, Murillo Rodrigues

Other: Josh Schiffman, Sergey Nuzhdin, Andy Kern

Funding: NSF ABI, Sloan, UO

Slides with [reveal.js](http://hakim.se) and [pandoc](https://pandoc.org/).


<!-- MISCELLANEA -->

##

miscellanea


Hitchhiking
-----------

![Maynard Smith & Haigh](figs/from_the_literature/maynard-smith-haigh-hitchhiking-dip.png)


Background selection
--------------------

![Hudson 1994](figs/from_the_literature/hudson-1994-background-dip-in-diversity.png)


Balanced polymorphism
---------------------

![Kreitman & Hudson 1991](figs/from_the_literature/kreitman-hudson-fig-5.png)


3. Recent balancing selection
----------------------

![](figs/linked-selection-guide/linked-selection-site-recent_balancing.png)


4. Long-term balancing selection
----------------------

![](figs/linked-selection-guide/linked-selection-site-old_balancing.png)


## *Drosophila melanogaster*

![McGaugh et al 2012](figs/from_the_literature/mcgaugh-2012-drosophila-dip.png)



---------------------

![](figs/aurantiacus/diversity-recomb.png)


------------------

![](figs/aurantiacus/correlations.png){width=70%}
